cmake/ActsComponentsHelpers.cmake

```diff
@@ -31,13 +31,11 @@ set(_components)
 function(add_component_if path name)
   file(RELATIVE_PATH _rel ${PROJECT_SOURCE_DIR} "${CMAKE_CURRENT_SOURCE_DIR}/${path}")
   if(${ARGN})
-    add_subdirectory(${path})
     list(APPEND _components "${name}")
     # propagate variable outside function scope
     set(_components "${_components}" PARENT_SCOPE)
     message(STATUS "Enable component '${name}' in '${_rel}'")
-  else()
-    message(STATUS "Disable component '${name}' in '${_rel}'")
+    add_subdirectory(${path})
```

Comment:
I think this output is really helpful. Can we keep it?

Reply:
Sure. It is back now.